src/test/java/com/mathworks/ci/MatlabBuilderTest.java

```diff
@@ -267,7 +267,7 @@ public void verifyCustomCommandInvoked() throws Exception {
         project.getBuildersList().add(this.matlabBuilder);
         FreeStyleBuild build = project.scheduleBuild2(0).get();
         jenkins.assertLogContains("-r", build);
-        jenkins.assertLogContains("try,runtests", build);
+        jenkins.assertLogContains("try,eval", build);
```

Comment:
I think it might be useful to try a few particular test cases for the custom string, like:   ") disp('hello') eval("